models/webhook.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Gogs Authors. All rights reserved.
+// Copyright 2017 Gitea. All rights reserved.
```

Comment:
Okay, I took the example from this file https://github.com/go-gitea/gitea/blob/master/models/status.go#L1